examples/gauging/gauging_itns.jl

```diff
@@ -7,38 +7,38 @@ using SplitApplyCombine
 
 using ITensorNetworks:
   message_tensors,
-  nested_graph_leaf_vertices,
-  belief_propagation_iteration,
   belief_propagation,
-  find_subgraph,
   vidal_gauge,
   symmetric_gauge,
   vidal_itn_canonicalness,
-  vidal_to_symmetric_gauge,
   initialize_bond_tensors,
   vidal_itn_isometries,
   norm_network,
-  edge_sequence
+  edge_sequence,
+  belief_propagation_iteration
 
 using NamedGraphs
 using NamedGraphs: add_edges!, rem_vertex!, hexagonal_lattice_graph
 using Graphs
 
 """Eager Gauging"""
-function eager_gauging(ψ::ITensorNetwork, bond_tensors::DataGraph, mts::DataGraph)
+function eager_gauging(
+  ψ::ITensorNetwork, pψψ::PartitionedGraph, bond_tensors::DataGraph, mts
```

Comment:
I think it's unfortunate that with this new design we know have even more inputs to this function, let's discuss that.